notebooks/README.md

```diff
@@ -0,0 +1,30 @@
+# Jupyter Notebooks
+
+Helpful notebooks for users and admins to do analysis and data management using the DIVE Web REST API.
```

Comment:
I added this analysis notebook because I think it will be helpful for folks to see how to get at their girder datasets from python.  The first example is pretty straightforward.

It needs a reference in the docs somewhere.